scripts/postInstall.js

```diff
@@ -0,0 +1,26 @@
+// @flow strict
+
+const fs = require('fs');
+const path = require('path');
+
+const log = message => console.log(`➤ ${message}`); // eslint-disable-line no-console
+
+log('Patching intl package');
```

Comment:
Maybe could you add a comment linking to a GitHub issue explaining why this patch is necessary?